<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>592</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Generateur de graphes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:#374d64;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,22">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,3,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QToolButton" name="toolButton_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">#toolButton_2{
        border:0;
        background-image: url(:/imgs/plus.png);
}

#toolButton_2:hover{
        background:#2c3843;
        border-radius:5px;
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/imgs/plus.png</normaloff>:/imgs/plus.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::NoArrow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QToolButton" name="random">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#random{
background:none;
background-color:transparent;
border:0;
}

#random:hover{
        background:#2c3843;
        border-radius:5px;
}
</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/imgs/random.png</normaloff>:/imgs/random.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QToolButton" name="save">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#save{
background:none;
background-color:transparent;
border:0;
}

#save:hover{
        background:#2c3843;
        border-radius:5px;
}
</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/imgs/save.png</normaloff>
                <normalon>:/imgs/edit.png</normalon>:/imgs/save.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QToolButton" name="reset">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#reset{
background:none;
background-color:transparent;
border:0;
}

#reset:hover{
        background:#2c3843;
        border-radius:5px;
}
</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/imgs/reset.png</normaloff>:/imgs/reset.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55557f;&quot;&gt;Supprimer un sommet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true">#toolButton{
background:none;
background-color:transparent;
border:0;
}

#toolButton:hover{
        background:#2c3843;
        border-radius:5px;
}</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="ressources.qrc">
                <normaloff>:/imgs/close_white.png</normaloff>:/imgs/close_white.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,11">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0,0,10">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:#FFFFFF;
font: 600 12pt &quot;Gotham&quot;;</string>
          </property>
          <property name="text">
           <string>Choose your algorithm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dijkstra">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0;
color:#FFF;
font: bold 10pt &quot;Gotham&quot;;
color:#2c3e50;
background-color: rgb(87, 255, 180);
border-radius:5px;</string>
          </property>
          <property name="text">
           <string>Dijkstra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bellman_kalaba">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0;
color:#FFF;
font: bold 10pt &quot;Gotham&quot;;
color:#2c3e50;
background-color: rgb(87, 255, 180);
border-radius:5px;</string>
          </property>
          <property name="text">
           <string>Bellman-Kalaba</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aStar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0;
color:#FFF;
font: bold 10pt &quot;Gotham&quot;;
color:#2c3e50;
background-color: rgb(87, 255, 180);
border-radius:5px;</string>
          </property>
          <property name="text">
           <string>A*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#2c3e50;
border:none;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
